### ISSUE TITLE
Checking if an asset is a device target and should NOT be send to cropping and derivative modules will be handled on integration server.

### ISSUE THREAD

**Issue by Baeist** [NHMDenmark/DaSSCo-Integration#111]
Checking if an asset is a device target and should NOT be send to cropping and derivative modules will be handled on integration server.

_Originally posted by @ThomasAlscher1991 in https://github.com/NHMDenmark/DaSSCo-Image-Refinery/issues/253#issuecomment-2295876533_
          
We want to remove the asset from the pipeline once it has been identified as a asset_subject: "device target" --- this should happen before the status of the job gets updated.

**Comment by ThomasAlscher1991** [comment]
Do you mean the job status of barcodeReader?

**Comment by Baeist** [comment]
Yes, this is so the asset wont get send to the next job in line before the jobs are eliminated from the job list.
